tests/jballerina-integration-test/src/test/resources/configurables/configLibProject/modules/mod1/lib.bal

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+public type Manager record {|
```

Comment:
Shall we check the other `reodonly` record cases as well? 
Similar to https://github.com/ballerina-platform/ballerina-lang/pull/29865/files#diff-cefc21eff80a1358a9212cd12d9a9276a8ff1f4137362ad55bf194ca8c725277

Reply:
Fixed